lib/reporter.js

```diff
@@ -14,9 +14,9 @@ module.exports = function () {
 			log('Flying with ' + fmt.path + '...', obj.path)
 		})
 
-		.on('flyfile_not_found', function (obj) {
+		.on('flyfile_not_found', function () {
 			process.exitCode = 1
-			log('No Flyfile Error: ' + fmt.error, obj.error)
+			log('Could not find flyfile in current directory')
```

Comment:
Message isn't quite right. `utils.read` will [traverse upwards](https://www.npmjs.com/package/find-up) looking for a flyfile.